packages/bootstrap-vue-next/src/composables/useModalController/index.ts

```
@@ -1,28 +1,136 @@
+      return {
+        modal,
+        show() {
+          modal?.promise.value.show()
+        },
+        hide(trigger?: string) {
+          modal?.promise.value.hide(trigger)
+        },
+      }
+    }
+    if (registry?.value) {
+      for (const [, modal] of registry?.value.entries()) {
```

Biome (1.9.4)

[error] Unsafe usage of optional chaining.

If it short-circuits with 'undefined' the evaluation will throw TypeError here:


(lint/correctness/noUnsafeOptionalChaining)